STYLEGUIDE.md

```diff
@@ -72,3 +76,74 @@ import 'camera.dart';
 import 'game.dart';
 import 'viewport.dart';
 ```
+
+### Identifiers
+
+Use all [effective dart patterns](https://dart.dev/guides/language/effective-dart)
+
+* Avoid using global constants, but if you do use, name it prefixing with a `k`.
+
+```dart
+// Do not:
+const someConstant = 42;
+
+// Do instead:
+const kSomeConstant = 42;
+```
+
+### Comments
+
+* Add an identifier for TODO comments
+* Use fully capitalized TODO notation
+
+```dart
+
+// Do not:
+// Todo: This thing should be that thing
+final thisThing = 13;
+
+// Do instead:
+// TODO(wolfenrain): This thing should be that thing
+const thisThing = 13;
+```
+
+### Asserts
+
+* Use asserts to detect contract violation.
+* Use asserts to run debug only code.
```

Comment:
also don't quite like this, seems hacky. I think we can leave it out until we ever need something like this and then we can discuss.

Reply:
done